### PR TITLE
Axit

### PR DESCRIPTION
**Pull request by KamilWasilonek**
https://kamilwasilonek.github.io/ui-internship-lwo2/html/Axit/

### CODE REVIEW

File: html/Axit/index.html [hunk 1]
```diff
@@ -0,0 +1,591 @@
...
+      <a href="#"><img src="./assets/socials-black/wifi-connection-signal-symbol (1).png" alt=""></a>
+      
+    </div>
+  </section>
+
+  <!-- Tabs -->
+  <section class="tabs">
+
+    <!-- Buttons -->
+    <div class="tabs__tab-container">
```
Comment (tkachukivan) [inline html/Axit/index.html]: also sometimes it is possible to do as list of buttons/links

File: html/Axit/index.html [hunk 2]
```diff
@@ -0,0 +1,591 @@
...
+    </div>
+
+    <!-- Text -->
+    <div class="sub-list__text">
+
+      <h2 class="title title--bold sub-list__title">
+        Sub list section
+      </h2>
+
+      <span class="separator sub-list__separator "></span>
```
Comment (tkachukivan) [inline html/Axit/index.html]: maybe before/after as it is decoration element

File: html/Axit/index.html [hunk 3]
```diff
@@ -0,0 +1,591 @@
...
+
+      <div class="sub-list__services">
+
+        <!-- Service 1 -->
+        <div class="sub-list-service">
+          <!-- Image -->
+          <img class="sub-list-service__icon" src="./assets/cloud-storage-icon/cloud-storage-download.png" alt="">
+      
+          <!-- Text -->
+          <div class="sub-list-service__text">
```
Comment (tkachukivan) [inline html/Axit/index.html]: details would be more describable than text

File: html/Axit/index.html [hunk 4]
```diff
@@ -0,0 +1,591 @@
...
+      <picture>
+        <source media="min-width:1024px" srcset="./assets/standard-picture/image-right-standard-u-262@3x.png">
+        <img src="./assets/standard-picture/image-right-standard-u-262@2x.png" alt="Some city view">
+      </picture>
+    </div>
+
+  </section>
+
+  <!-- Awsome things -->
+  <section class="awsome-things">
```
Comment (tkachukivan) [inline html/Axit/index.html]: awEsome

File: html/Axit/styles/style.scss [hunk 5]
```diff
@@ -0,0 +1,889 @@
...
+
+    &__mobile-btn {
+      display: none;
+    }
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .navbar {
+    &__menu {
```
Comment (tkachukivan) [inline html/Axit/styles/style.scss]: maybe there is a reason to write just `navbar__menu`

File: html/Axit/styles/style.scss [hunk 6]
```diff
@@ -0,0 +1,889 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,700&display=swap');
```
Comment (tkachukivan) [inline html/Axit/styles/style.scss]: it would be a bit easier to work with this if separate to a few files

File: html/Axit/index.html [hunk 7]
```diff
@@ -0,0 +1,591 @@
...
+  <title>Document</title>
+  <link rel="stylesheet" href="./styles/style.css">
+</head>
+<body>
+
+  <!-- Banner -->
+  <section class="banner">
+
+    <!-- Navbar -->
+    <header class="navbar banner__navbar">
```
Comment (tkachukivan) [inline html/Axit/index.html]: this header is not related to section and can be by itself

File: html/Axit/index.html [hunk 8]
```diff
@@ -0,0 +1,591 @@
...
+      <span class="separator pricing__separator "></span>
+
+      <p class="description pricing__description">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      </p>
+
+      <div class="pricing__prices">
+
+        <!-- Price 1 -->
+        <button class="price pricing__price">
```
Comment (tkachukivan) [inline html/Axit/index.html]: I'm not sure if it button, as probably it is not and it is just a block, generally your button will have no other tags, maybe some spans

File: html/Axit/index.html [hunk 9]
```diff
@@ -0,0 +1,591 @@
...
+          Lorem ipsum dolor sit amet, 
+          consectetur adipisicing elit. Doloribus 
+          accusamus expedita repellat similique 
+          odio aspernatur ex, architecto eaque 
+          quo suscipit.
+          <span class="customer__indicator"></span>
+        </p>
+        <div class="customer__details">
+          <img src="./assets/customer/avatar-2-u-231@3x.png" alt="Jermey H.">
+          <div class="customer__names">
```
Comment (tkachukivan) [inline html/Axit/index.html]: not sure if customer position is part of his name, maybe some more general class name would be better here

File: html/Axit/index.html [hunk 10]
```diff
@@ -0,0 +1,591 @@
...
+      <div class="contact__inputs">
+        <div class="contact__inputs-left">
+          <input class="form__input contact__input" type="text" placeholder="Name">
+  
+          <input class="form__input contact__input" type="email" placeholder="Email">
+  
+          <input class="form__input contact__input" type="text" placeholder="Subject">
+        </div>
+
+        <input class="form__input contact__message" type="text" placeholder="Message">
```
Comment (tkachukivan) [inline html/Axit/index.html]: it is textarea not input

**Review by tkachukivan — CHANGES_REQUESTED** [review]
It would be great to change a bit inputs on focus, for example change border color

**Review by tsvetkovv — APPROVED** [review]

File: .gitignore [hunk 11]
```diff
@@ -3,6 +3,8 @@
 .vscode/
 *.iml
 *.sublime*
+*.css
```
Comment (tkachukivan) [inline .gitignore:6]: i hope you will not have conflict with other branch

**Review by tkachukivan — APPROVED** [review]

**Pull request merged**